site/content/docs/12-security.md

```diff
@@ -6,9 +6,9 @@ By default, Sapper does not add security headers to your app, but you may add th
 
 ### Content Security Policy (CSP)
 
-Sapper generates inline `<script>`s, which can fail to execute if [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers disallow arbitrary script execution (`unsafe-inline`).
+Sapper generates inline `<script>`s and `<style>`s, which can fail to execute if [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers disallow arbitrary script execution (`unsafe-inline`) or style parsing (`unsafe-inline`).
```

Comment:
This says `unsafe-inline` twice. Are there different CSP values for disallowing arbitrary script execution and arbitrary style parsing?

Reply:
The CSP can have both `script-src` and `style-src` directives with `unsafe-inline` being a valid source for both. 

In this context I think it would make more sense to remove the ```(`unsafe-inline`)``` text. I've updated the PR.